src/frontend/styles/_buttons.scss

```diff
@@ -40,33 +40,43 @@ button {
 }
 
 .button.is-primary {
-  background: var(--primary);
+  background: var(--primary-button, var(--primary));
 }
 
 .button.is-primary:hover {
-  background: var(--primary-hover);
+  background: var(--primary-button, var(--primary-hover));
```

Comment:
if `--primary-button` is defined then the hovered and non-hovered backgrounds will be the same

Reply:
That's intentional, on the new theme now we add a Box shadow on the primary and success buttons instead of changing the background. It is a way of differentiating from the install button. It was not my idea, it is on the new design system on figma for the next UI but I think it is cool so I added it here on this design as well.
I think Billie changed that because for some people the color of the button won't change due to visual impairment. I have some issues myself with colors that are too close to each other so adding this box shadow, makes it is easier to see that we are hovering it.

![image](https://user-images.githubusercontent.com/26871415/216041955-fd4034df-3fd1-4d4d-8f80-8fd3db26347a.png)

![image](https://user-images.githubusercontent.com/26871415/216042022-f833dbfe-56e8-4c67-ae82-cf315f95fcb8.png)